extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/internal/config/WeatherUndergroundConfiguration.java

```diff
@@ -15,6 +15,9 @@
  */
 public class WeatherUndergroundConfiguration {
 
+    public static final String APIKEY = "apikey";
```

Comment:
It was at a certain of this PR. I kept it thinking it could be used one day later. But I can suppress it.